content/release/8.4.5.md

```diff
@@ -0,0 +1,93 @@
+---
+Title: FRRouting Release 8.4.5
+Release: 8.4.5
+GitHub: https://github.com/FRRouting/frr/releases/tag/frr-8.4.5
+Date: 2024-06-06
+---
+
+FRRouting Release 8.4.5
+=======================
+
+## Fixed CVEs
+- [CVE-2024-31950](https://frrouting.org/security/cve-2024-31950)
+- [CVE-2024-31951](https://frrouting.org/security/cve-2024-31951)
+- [CVE-2023-38802](https://frrouting.org/security/cve-2023-38802)
+- [CVE-2023-46752](https://frrouting.org/security/cve-2023-46752)
+- [CVE-2023-46753](https://frrouting.org/security/cve-2023-46753)
+- [CVE-2023-47235](https://frrouting.org/security/cve-2023-47235)
+- [CVE-2024-31948](https://frrouting.org/security/cve-2024-31948)
+
+## Bug Fixes
+
+babeld
+* Fix #11808 to avoid infinite loops
```

Comment:
This link leads to nowhere.

Reply:
This isn't a link, just a PR number, whatever was there in the original PR comment